src/core/background/main.ts

```diff
@@ -272,6 +273,53 @@ setupBackgroundListeners(
 		},
 	}),
 
+	/**
+	 * Listener called by a controller to trigger sending of now playing info.
+	 */
+	backgroundListener({
+		type: 'setNowPlaying',
```

Comment:
I feel like that might be a bit overkill, since the type itself already restricts the strings.
As long as youre using an IDE with decent TS support, it should already be suggesting/autocompleting existing ones and erroring if it doesnt exist.

Reply:
More thinking of typos and with TS you'll get compilation error on such case I'd assume :)

Reply:
Indeed, typos lead to compilation error already

Reply:
Interesting - I see what you mean - VSCode makes it run but compilations still work :/ am I doing something wrong?

Reply:
Vite assumes your IDE takes care of type checking, and skips this to save time.
It only compiles, and assumes the types are good.
We could potentially add a type checking step somewhere, but thatd be out of scope for this PR.

Reply:
For sure out of scope :)